src/opt/RemovePersistentFuncs.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+  Copyright (c) 2022-2024, Intel Corporation
```

Comment:
NIT: Looks like it needs to be just `2024`. It's a new file, right?

Reply:
Right, it is just a copy-paste leftover.